test/suites/mask_fromjson.yml

```
@@ -2,37 +2,78 @@ name: fromjson features
+          version: "1"
+          masking:
+            - selector:
+                jsonpath: "targetfield"
+              mask:
+                fromjson: "sourcefield"
+          EOF
           echo '{"sourcefield": "null", "targetfield": ""}' | pimo
         assertions:
           - result.code ShouldEqual 0
-          - result.systemout ShouldEqual {"sourcefield": "null", "targetfield": null}
+          - result.systemout ShouldEqual {"sourcefield":"null","targetfield":null}
           - result.systemerr ShouldBeEmpty
   - name: integer value
     steps:
-      - script: cp ../masking.yml masking.yml
+      - script: rm -rf masking.yml
+      - script: |-
+          cat > masking.yml <<EOF
+          version: "1"
+          masking:
+            - selector:
+                jsonpath: "targetfield"
+              mask:
+                fromjson: "sourcefield"
+          EOF
+          echo '{"sourcefield": "null", "targetfield": ""}' | pimo
+        assertions:
       - script: |-
           echo '{"sourcefield": "1", "targetfield": ""}' | pimo
         assertions:
           - result.code ShouldEqual 0
-          - result.systemout ShouldEqual {"sourcefield": "1", "targetfield": 1}
+          - result.systemout ShouldEqual {"sourcefield":"1","targetfield":1}
           - result.systemerr ShouldBeEmpty
   - name: float value
     steps:
-      - script: cp ../masking.yml masking.yml
+      - script: rm -rf masking.yml
+      - script: |-
+          cat > masking.yml <<EOF
+          version: "1"
+          masking:
+            - selector:
+                jsonpath: "targetfield"
+              mask:
+                fromjson: "sourcefield"
+          EOF
+          echo '{"sourcefield": "null", "targetfield": ""}' | pimo
+        assertions:
       - script: |-
           echo '{"sourcefield": "1.2", "targetfield": ""}' | pimo
         assertions:
           - result.code ShouldEqual 0
-          - result.systemout ShouldContainSubstring {"sourcefield": "1.2", "targetfield": 1.2}
+          - result.systemout ShouldContainSubstring {"sourcefield":"1.2","targetfield":1.2}
           - result.systemerr ShouldBeEmpty
   - name: float value
     steps:
-      - script: cp ../masking.yml masking.yml
+      - script: rm -rf masking.yml
+      - script: |-
+          cat > masking.yml <<EOF
+          version: "1"
+          masking:
+            - selector:
+                jsonpath: "targetfield"
+              mask:
+                fromjson: "sourcefield"
+          EOF
+          echo '{"sourcefield": "null", "targetfield": ""}' | pimo
+        assertions:
       - script: |-
           echo '{"sourcefield": "{\"property\": \"hello\"}", "targetfield": ""}' | pimo
         assertions:
           - result.code ShouldEqual 0
-          - result.systemout ShouldEqual {"sourcefield": "{\"property\": \"hello\"}", "targetfield": "hello"}
+          - result.systemout ShouldEqual {"sourcefield":{"property":"hello"},"targetfield":"hello"}
```

Non ce n'est pas exactement ce que j'ai demandé.

Voici la bonne version

```suggestion
          - result.systemout ShouldEqual {"sourcefield":"{\"property\":\"hello\"}","targetfield":{"property":"hello"}}
```